pkg/gui/presentation/branches.go

```diff
@@ -79,19 +82,26 @@ func GetBranchTextStyle(name string) style.TextStyle {
 
 func ColoredBranchStatus(branch *models.Branch) string {
 	colour := style.FgYellow
-	if branch.MatchesUpstream() {
-		colour = style.FgGreen
-	} else if !branch.IsTrackingRemote() {
+	if !branch.IsTrackingRemote() || branch.Gone {
 		colour = style.FgRed
+	} else if branch.MatchesUpstream() {
+		colour = style.FgGreen
 	}
 
 	return colour.Sprint(BranchStatus(branch))
 }
 
 func BranchStatus(branch *models.Branch) string {
```

Comment:
we'll need to pass in that tr object here too. Annoying because we're calling this function from another file. But I'm happy for us to do that even if it makes things a little hairier until a proper refactor comes later.